packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -125,8 +125,6 @@
 
 /// @arg $type Can be one of: expander, status
 @mixin iui-expandable-block-header-icons($type) {
-  @include mixins.iui-icon-style('m');
```

Comment:
below, instead of setting `inline-size`, `block-size` and `fill`, you can set these css variables.

```scss
--iui-svg-size: var(--_iui-expandable-block-expander-icon-size);
--iui-svg-fill: var(--_iui-expandable-block-expander-icon-fill);
```

Reply:
thanks! this works great for adjusting the icon size based on attribute. I was going to go `:where` route..